### PR TITLE
feat: authentication pages added

### PR DESCRIPTION
**Pull request by mdarif6**
authentication pages added

### CODE REVIEW

File: src/login-page/LoginMain.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        password: password,
+      });
+
+      if (response.status === 200 || response.status === 201) {
+        localStorage.setItem("authToken", response.data.encodedToken);
+        navigate("/");
+        dispatch({ type: "SET_AUTH", payload: true });
+      }
+    } catch (error) {
+      console.log(error);
```
Comment (anuj-kumary) [inline src/login-page/LoginMain.jsx:30]: You can show errors using the **console.error**.

File: src/login-page/LoginMain.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    }
+  }
+
+  async function loginAsGuest() {
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: "adarshbalika@gmail.com",
+        password: "adarshBalika123",
+      });
+      console.log(response);
```
Comment (anuj-kumary) [inline src/login-page/LoginMain.jsx:40]: **suggestion**: delete console logs if not needed in future

File: src/login-page/LoginPage.css [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  background-color: rgb(58, 52, 52);
+}
+.outline-primary {
+  margin: 0 1rem;
+  width: 90%;
+  color: gray !important;
+}
+.outline-primary:hover {
+  background-color: rgb(46, 43, 43) !important;
+  color: white !important;
```
Comment (anuj-kumary) [inline src/login-page/LoginPage.css:46]: **Suggestion**: Don't use !**important**

**Review by anuj-kumary — APPROVED** [review]
Clean code and UI
Good to merge

File: src/login-page/LoginPage.css (hunk 3 above)
Comment (mdarif6) [reply src/login-page/LoginPage.css:46]: - yes u r right but i don't know how to override previous style which is coming from my component library

File: src/login-page/LoginMain.jsx (hunk 2 above)
Comment (mdarif6) [reply src/login-page/LoginMain.jsx:40]: sure

**Review by SJTGSHIVAM — APPROVED** [review]
Changes looks good just remember to remove console logs

**Pull request merged**